org.eclipse.winery.repository.ui/src/app/wineryInterfaces/enums.ts

```diff
@@ -45,3 +45,12 @@ export enum ServiceTemplateTemplateTypes {
     RelationshipTemplate = 'relationshipTemplates',
     RequirementTemplate = 'requirementTemplates'
 }
+
+/**
+ * BackendStates defines the state for the availability of the backend.
+ */
+export enum BackendStates {
+    Available = 1,
```

Comment:
Is there a naming convention for ENUMs? In Java, they are written upper case.

Reply:
Yes, in TypeScript, you use PascalCase, see [Microsofts Coding Guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names) or the offical [TypeScript Documentation](https://www.typescriptlang.org/docs/handbook/enums.html)